test/lib/server.test.js

```diff
@@ -33,35 +27,29 @@ describe('server case', () => {
     });
 
     describe('positive cases', () => {
+        let registrationManMock;
+
         beforeEach(() => {
+            registrationManMock = sinon.stub();
+
+            mockery.registerMock('./registerPlugins', registrationManMock);
             /* eslint-disable global-require */
             hapiEngine = require('../../lib/server');
             /* eslint-enable global-require */
         });
 
-        it('injects the status', (done) => {
-            hapiEngine((error, server) => {
-                Assert.notOk(error);
-                server.inject({
-                    method: 'GET',
-                    url: '/v3/status'
-                }, (response) => {
-                    Assert.equal(response.statusCode, 200);
-                    done();
-                });
-            });
-        });
-
         it('does it with a different port', (done) => {
-            processEnvMock.PORT = 12347;
+            registrationManMock.yieldsAsync(null);
 
-            hapiEngine((error, server) => {
+            hapiEngine({
+                port: 12347
+            }, (error, server) => {
                 Assert.notOk(error);
                 server.inject({
                     method: 'GET',
-                    url: '/v3/status'
+                    url: '/blah'
```

Comment:
because we mock out registerMan, so it doesn't really register the plugin and so this path doesn't work.